### PR TITLE
Changes start time source and prevents duplicate logs

### PR DESCRIPTION
**Pull request by PGuimarais**
- Prevents duplicate logs from loading by changing infinite scroll conditions
- Changes default values in Redux store

Fixes:
https://issues.cask.co/browse/CDAP-7138
https://issues.cask.co/browse/CDAP-7121
https://issues.cask.co/browse/CDAP-7123

Bamboo Build:
http://builds.cask.co/browse/CDAP-DRC4352

### CODE REVIEW

File: cdap-ui/app/directives/log-viewer/log-viewer-store.js [hunk 1]
```diff
@@ -15,30 +15,44 @@
...
         return action.payload.startTime;
       case LOGVIEWERSTORE_ACTIONS.RESET:
-        return new Date();
+        return null;
+      default:
+        return state;
+    }
+  };
+
+  const globalStartTime = (state = null, action = {}) => {
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/log-viewer/log-viewer-store.js:33]: I am not sure if I understand the difference between startTime and globalStartTime. Probably a short comment in the code and a brief writeup in the PR?

File: cdap-ui/app/services/auth.js [hunk 2]
```diff
@@ -38,34 +38,34 @@ module.constant('MYAUTH_ROLE', {
 });
 
 
-module.run(function ($location, $state, $rootScope, myAuth, MYAUTH_EVENT, MYAUTH_ROLE) {
+// module.run(function ($location, $state, $rootScope, myAuth, MYAUTH_EVENT, MYAUTH_ROLE) {
```
Comment (ajainarayanan) [inline cdap-ui/app/services/auth.js]: This should not be checked in right? We talked about this

File: cdap-ui/server/config/development/cdap.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "stream.index.interval": "10000",
+  "stream.index.interval": "11015",
```
Comment (ajainarayanan) [inline cdap-ui/server/config/development/cdap.json]: This either. This is already changed in develop I believe.

File: cdap-ui/server/config/ui-settings.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
-  "standalone.welcome.message": "true",
+  "standalone.welcome.message": "false",
```
Comment (ajainarayanan) [inline cdap-ui/server/config/ui-settings.json]: This should not be changed either. The default is to show welcome screen for now.

**Comment by ajainarayanan** [comment]
@PGuimarais View raw logs and Download logs are not working properly. The start time we generate for those request seems to be wrong.

**Comment by PGuimarais** [comment]
Will be revisiting this later

**Pull request closed without merging**